rslib/src/search/mod.rs

```diff
@@ -192,11 +174,11 @@ impl Collection {
         if mode == &SortMode::FromConfig {
             *mode = match items {
                 SearchItems::Cards => SortMode::Builtin {
-                    kind: self.get_browser_sort_kind(),
+                    column: self.get_browser_sort_column(),
```

Comment:
Perhaps "SearchContext", "SearchScope", "SearchMode" or similar might be slightly better than "SearchItems"?

Reply:
I don't mind changing the name but I have trouble with these particular suggestions. SearchMode sounds good, but `mode` is already used as a paramater name for SortMode. SearchContext sounds a bit generic given the importance of the enum to the search behaviour whereas SearchScope sounds as if only Cards *or* Notes where searched (when we actually search both but only return one type). I currently don't have a better idea, though.
By the way, the name SearchItems was inspired by the ItemId on the frontend.

Reply:
ReturnType? ReturnItemType? ItemType?